src/test/resources/org/jenkinsci/plugins/xunit/types/valgrind/testcase3/input.xml

```diff
@@ -0,0 +1,167 @@
+<?xml version="1.0"?>
```

Comment:
TC 3 and 4 are 99% the same - just keep one of them
please use "anonymized" version from PR #94:
https://github.com/jenkinsci/xunit-plugin/pull/94/files#diff-876faac5d388eb738cd0c795eb4cbc26fb5c95d81ba1b59f14b207a51d362cd5

Reply:
this get when I see your PR, any way this pr is just a demostration for comment in your PR. I will close this one in favor of yours

Reply:
naah, your approach seems more complex.